src/OpenTelemetry.Extensions/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Add LogToActivityEventConversionOptions.Filter callback
```

Comment:
it'd be good to modify the readme.md to document this. good for a follow up PR as the current readme is also missing any docs